### DISCUSSION TITLE
Help with my remote for my TV

### DISCUSSION THREAD

**Discussion by fsamson91** [ts1/PiIR#8] [Q&A]
Hello, I tried  your PiiR library to duplicate some remote for my TV, etc.. I use the piir program to record, so this part work fine I have a json file with different IR command but when I play with it nothing on my TV. 
So I try to play and record at the same time and the new json file is the same... I try at 5cm of the TV and nothing... 
Do you have any idea to help me ? 
Thanks

**Comment by ts1** [comment]
It seems like a hardware issue.
If you look with the webcam or the selfie camera on an iPhone, you can see the IR LEDs emitting light.
With the iPhone's rear camera you can't see it because of the IR filter in the camera.
I think it varies for Android, but I don't know much about them.

If you see the IR on your remote control, look at the Pi's IR LED and check to see if it is emitting light.

**Comment by fsamson91** [comment]
At the camera The emitter send very low light than the real TV remote command... So I order an another IR emitter to test. Thanks for your help.

**Reply by ts1** [reply]
The amount of current flowing through it is just as important as the LED itself.
